core/domain/suggestion_registry.py

```diff
@@ -654,6 +724,48 @@ def _get_skill_difficulty(self):
         """Returns the suggestion's skill difficulty."""
         return self.change.skill_difficulty
 
+    def get_all_html_content_strings(self):
+        """Gets all html content strings used in this suggestion.
+
+        Returns:
+            list(str). The list of html content strings.
+        """
+        html_string_list = []
+        interaction_object = (
+            state_domain.InteractionInstance.from_dict(
+                self.change.question_dict['question_state_data'][
```

Comment:
Why not use the method in the State class? (Note: question_state_data is going to be  state dict)

Reply:
done